chefignore

```diff
@@ -0,0 +1,9 @@
+*/.chef
```

Comment:
Shouldn't this file be named `.chefignore` (https://docs.chef.io/chef_repo.html#chefignore-files) or is the leading dot optional?